flexget/plugins/filter/upgrade.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *
+from collections import defaultdict
+from datetime import datetime
+from sqlalchemy import Column, String, Unicode, DateTime, Boolean
+import logging
+
+from flexget import db_schema, plugin
+from flexget.entry import Entry
+from flexget.utils.database import quality_property
+from flexget.db_schema import Session
+from flexget.event import event
+from flexget.utils import qualities
+
+log = logging.getLogger('upgrade')
+
+Base = db_schema.versioned_base('upgrade', 0)
+
+entry_actions = {
+    'accept': Entry.accept,
+    'reject': Entry.reject,
+    'fail': Entry.fail
+}
+
+
+class EntryUpgrade(Base):
+    __tablename__ = 'upgrade'
+
+    id = Column(Unicode, primary_key=True, index=True)
+    title = Column(Unicode)
+    _quality = Column('quality', String)
+    quality = quality_property('_quality')
+    proper = Column(Boolean)
+    added = Column(DateTime, index=True)
```

Comment:
you can set `default=datetime.now()` here and not handle in via the plugin

Reply:
ah missed that